src/impl/english.js

```diff
@@ -73,58 +74,56 @@ export class English {
       case 'long':
         return English.weekdaysLong;
       case 'numeric':
-        return ['1', '2', '3', '4', '5', '6', '7'];
       default:
-        return null;
+        return ['1', '2', '3', '4', '5', '6', '7'];
     }
   }
 
-  static get meridiems() {
+  static get meridiems(): Array<string> {
     return ['AM', 'PM'];
   }
 
-  static get erasLong() {
+  static get erasLong(): Array<string> {
     return ['Before Christ', 'Anno Domini'];
   }
 
-  static get erasShort() {
+  static get erasShort(): Array<string> {
     return ['BC', 'AD'];
   }
 
-  static get erasNarrow() {
+  static get erasNarrow(): Array<string> {
     return ['B', 'A'];
   }
 
-  static eras(length) {
+  static eras(length: string): Array<string> {
     switch (length) {
       case 'narrow':
         return English.erasNarrow;
       case 'short':
         return English.erasShort;
       case 'long':
-        return English.erasLong;
       default:
-        return null;
+        return English.erasLong;
     }
   }
 
-  static meridiemForDateTime(dt) {
+  static meridiemForDateTime(dt: Object) {
```

Comment:
`Object` is as typesafe as `any`. Could you use a `*` or create a interface that could be built on later?

Reply:
I think the plan would be create a datetime type somewhere and import it. My intention with the PR was to try to get the ball rolling ... I didn't necessarily want to create definitions, just get the pipeline working so that adding defs could be a community effort. If the maintainer(s) feel differently I can of course expand on this ...

Reply:
I'm good with getting something simple in and tightening it down later. My worry with this as it stands now is that there's not enough of it to make a foothold. I don't want there to be like a random scattering of types. So I'm hoping someone will take this and add a bunch of stuff in before I do any merging.

Reply:
Do you want people to add types to this PR then?
I could probably spend some time on it although I actually want it in a typescript project. Or maybe this is a good reason to ditch TS and use flow instead 🤔

Reply:
I think adding types to this PR probably makes sense.

Reply:
I'll keep plugging away at it when I have a chance, but unfortunately got the flu last week and will be in catchup mode for a while. If other people want to add to it or re-fork it it's totally fine with me.

Reply:
@ulrikstrid I remember the author of the React Material UI project said one of the things he learned from writing it is that a lot of people are moving from Flow to TypeScript.  I'm not 100% sure why but I get the impression TypeScript is probably more stable and has more convenient error messages (though I'm a Flow user and have never used TS myself).  So definitely don't hurry the decision to switch to Flow.

Reply:
I have used both and I love the number of typed packages for TypeScript but Flow feels so much better. If there were a larger community around Flow where more typings for packages I would love to use that always.
Also as a consultant I sometimes have to make the "safe bet" and that is still TS these days.

Reply:
Huh, what feels better about it?
As far as staying on topic, I'm going to try to PR typedefs for `luxon` that I've been using locally to `flow-typed` today.

Reply:
@ulrikstrid btw one very cool thing you may want to check out, which I don't think is currently possible with TS, is [flow-runtime](https://github.com/codemix/flow-runtime).